README.md

```diff
@@ -15,6 +15,9 @@ Flintrock has been featured in a few talks, guides, and papers around the web.
 * Talks:
   * [Flintrock: A faster, better spark-ec2](https://www.youtube.com/watch?v=3aeIpOGrJOA) ([slides](http://www.slideshare.net/SparkSummit/flintrock-a-faster-better-sparkec2-by-nicholas-chammas))
 * Guides:
+  * **Running Spark on a Cluster: The Basics (using flintrock)**
```

Comment:
For brevity, can we just have this line in regular font and add "(part 1 / part 2)" at the end? With "part 1" and "part 2" linking to the two parts, respectively.

Also, please capitalize Flintrock.

Reply:
I'd rather not because the two guides are very different.

One teaches you how to use spark-shell. One teaches you how to actually package up real (not-totally-simple) pipelines and run them on a Spark cluster. 

I think that distinction should be made. They're really two separate guides under one theme; we use Flintrock instead of spark-ec2 to do two different things.

Reply:
Understood. To clarify my thought process, the reason I ask for this change is two-fold:
1. I'm trying not to elevate any one guide above the others by granting it more real estate in such a prominent place in the README.
2. The faster this section of the README grows, the sooner I will likely move it out of the README (and likely into the Wiki). Perhaps I should have done this from the start, since it reduces the pressure on me to curate references and gives us more freedom to expand on each reference.

That said, we can leave things as you have them now but in the future I may edit or move the references section for the above reasons.

Reply:
It's not particularly important to me if you want to "promote" what I've written or not.

I'm a professor teaching Spark to students. The documentation out there for submitting a Spark job to a cluster is really not great. For Flintrock or spark-ec2 or any other tool for that matter.

Since I've used Flintrock in my class, and since there is not much in the way of documentation for your project, I merely offer the guide to you in case you would like to use it in place of maintaining documentation yourself.

How and whether you manage your project is up to you! Just note that now that spark-ec2 is deprecated, I imagine you're going to have many more people looking for documentation/guides/etc on setting up such pipelines...

Reply:
I 100% appreciate the guides that you and others have written and believe they help close the gap between having a tool that works and having a tool that people feel comfortable using. I'm happy to link to them from the project and think they are helpful.

I'm just thinking out loud as a maintainer about how I'd like to not be in the business of curating external references. It's not about any individual guide. So for example, the note about wanting the reference to be limited to one line was motivated by a hypothetical future scenario where another guide author messages me, upset, that I've given guide X more prominence than their guide Y. And the note about potentially moving references to the Wiki was motivated by another hypothetical where more and more guide authors want to add links to their work, making the README a bit unwieldy.

It's silly, but that's the kind of stuff I worry about. 😄 I hope that clarifies my intentions. I hadn't really thought through how to manage the references section (which I only recently added to the README), so this PR happened to provide me the opportunity to hash my thought process out. Please accept my apologies for turning a simple PR into a lengthy discussion. 😅